modules/programs/neovim.nix

```diff
@@ -138,11 +138,14 @@ in {
       };
 
       withNodeJs = mkOption {
-        type = types.bool;
+        type = types.nullOr types.bool;
         default = false;
         description = ''
           Enable node provider. Set to <literal>true</literal> to
           use Node plugins.
+
+          This will be automatically enabled if <varname>coc.enable</varname> is set,
+          which can be circumvented by setting this option to <literal>null</literal>.
```

Comment:
what's the point of having this to null and coc.enable true if it doesn't bring nodejs ? shouldn'it be an error instead ?

Reply:
With that you can supply an older version of nodejs yourself using `extraPackages` which coc will pick up.